var/spack/repos/builtin/packages/f18/package.py

```diff
@@ -17,7 +17,3 @@ class F18(CMakePackage):
     version('develop', branch='master')
```

Comment:
You can actually rename this version to `master` now that #1983 has been merged.

Reply:
@certik @adamjstewart Sorry, I am not familiar with this package. I get the jist of the `Version` pull request, but did not follow the details.

You are saying to change the line to?

`version('master')` or

`version('master', branch='master')`

Reply:
The latter